ports/gtest/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "gtest",
-  "version-date": "2021-05-07",
+  "version": "1.11.0",
```

```suggestion
  "version-semver": "1.11.0",
```
It should be `version-semver `here.